lib/cripto_details/widgets/graphic.dart

```diff
@@ -84,11 +109,11 @@ class _GraphicState extends ConsumerState<Graphic> {
                   ),
                 ),
               ),
-              minX: widget.days - 1,
-              maxX: -0.2,
-              baselineX: 3,
-              minY: 0,
-              maxY: 103612.60,
+              // minX: widget.days - 1,
+              // maxX: -0.2,
+              // baselineX: 3,
+              // minY: 0,
+              // maxY: 103612.60,
```

Comment:
se não for relevante deixar documentado, da pra apagar tb

Reply:
feito, os comentários foram apagados